includes/class-wc-payments-payment-method-messaging-element.php

```diff
@@ -65,8 +79,10 @@ public function init(): string {
 			'WCPAY_PRODUCT_DETAILS',
 			'wcpayStripeSiteMessaging',
 			[
-				'price'          => WC_Payments_Utils::prepare_amount( $price, $currency_code ),
+				'price'          => $base_product_price,
 				'currency'       => $currency_code,
+				'productPrices'  => $product_prices,
+				'country'        => $billing_country,
```

Comment:
I think it's a merge-conflict artifact here.

Reply:
Sorry.. fixed here - 6deb419